src/cloudlabs/models.py

```diff
@@ -87,9 +87,9 @@ def __repr__(self):
 
     def __init__(self, *args, **kwargs):
         """Define a default setup_script as well as the supplied fields."""
-        if ('setup_script' not in kwargs and 'git_repo' in kwargs
-            and 'port' in kwargs):
-            kwargs['setup_script'] = self.default_setup_script(**kwargs)
+        #if ('setup_script' not in kwargs and 'git_repo' in kwargs
+        #    and 'port' in kwargs):
+        #    kwargs['setup_script'] = self.default_setup_script(**kwargs)
         super(Host, self).__init__(*args, **kwargs)
 
     def default_setup_script(self, **kwargs):
```

Comment:
You can also remove this. It was just needed for when we had no script in the form.